src/webgl/light.js

```diff
@@ -494,6 +494,12 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
   return this;
 };
 
+p5.prototype.imageLight = function(img){
+  // setting activeImageLight so that the setUniform is executed
```

Comment:
Maybe we can be a bit more specific here and say that this is what `_setImageLightUniforms` looks at to see if it needs to send uniforms to the fill shader